README.md

```diff
@@ -2,3 +2,16 @@
 This is my 1st GitHub repository.
 
 There's not much to see here, this is basically a "Hello World" repository.
+
+This is a change made on the "readme-edits" branch, as it is was supposed to be.
+
+* Note to self #1: NEVER commit on the "master" branch, always use a feature branch instead and then merge it.
+
+* Another oddity of Git is that feature branches are not useful anymore after they are merged using "pull-requests", they can be safely deleted, and the website prompts to do that (at some point).
+
+Task list:
+- Should delete unused/merged feature branches
+- Perform the same operations from the CLI/GitBash prompt
```

Comment:
This comment relates to 3 lines.
Added the "Task list" in the README.md file.

Reply:
Added yet another review comment.

Now the question arises, how does the review proces work.